storage/responses/request.index.json

```diff
@@ -0,0 +1,53 @@
+[
```

Comment:
Folder should be in `.gitignore`

Reply:
Why?

Reply:
Oh, my bad, I didn't realise they were examples. Ignore this.